### PR TITLE
Feature/course 1

### PR DESCRIPTION
**Pull request by lucalm91**

### CODE REVIEW

File: staff/luca-lamorte/feature/course-1/NYSL/home.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>North Side Youth Soccer League</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <header class="header">
+        <div class="hero">
```
Comment (manuelbarzi) [inline staff/luca-lamorte/feature/course-1/NYSL/home.html:15]: is this wrapping div (hero) necessary?

File: staff/luca-lamorte/feature/course-1/NYSL/home.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    <div class="main-container">
+        <nav class="navbar">
+            <ul>
+                <li><a href="home.html">Home</a></li>
+                <li><a href="about-us.html">About</a></li>
+                <li><a href="contact.html">Contact</a></li>
+            </ul>
+        </nav>
+
+        <div class="events">
```
Comment (manuelbarzi) [inline staff/luca-lamorte/feature/course-1/NYSL/home.html:34]: may this be a section?

File: staff/luca-lamorte/feature/course-1/NYSL/home.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            <div class="event-container">
```
Comment (manuelbarzi) [inline staff/luca-lamorte/feature/course-1/NYSL/home.html:35]: may you list events in a ul element?

**Review by manuelbarzi — CHANGES_REQUESTED** [review]

File: staff/luca-lamorte/feature/course-1/NYSL/about-us.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>North Side Youth Soccer League</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <header class="header">
+        <div class="hero">
```
Comment (RoseKoning) [inline staff/luca-lamorte/feature/course-1/NYSL/about-us.html:15]: You might not need a div for a <h1> element in a header

File: staff/luca-lamorte/feature/course-1/NYSL/about-us.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    <div class="main-container">
+        <nav class="navbar">
+            <ul>
+                <li><a href="home.html">Home</a></li>
+                <li><a href="about-us.html">About</a></li>
+                <li><a href="contact.html">Contact</a></li>
+            </ul>
+        </nav>
+
+        <div class="about-us-container">
```
Comment (RoseKoning) [inline staff/luca-lamorte/feature/course-1/NYSL/about-us.html:35]: this could also be a < section> element

**Comment by manuelbarzi** [comment]
There has been a change in the `upstream` repo, please, re-do this PR against your new branch `develop-<your-program-year-month`>. Thank you!

**Pull request closed without merging**